### PR TITLE
Newsletters: add 265 (2023-08-23)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/services section @bitschmidty 
- [x] Bitcoin Core 27460 @murchandamus

### CODE REVIEW

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  op_checksigfromstack] opcodes were added to Bitcoin in a soft fork, it
+  would be possible to use the fraud proof onchain.
+
+  For example, Alice and Bob share an LN channel which includes an
+  additional [taproot][topic taproot] condition that allows Alice to
+  spend all of the channel funds if she can provide this type of fraud
+  proof.  The normal operation of the channel would include just one
+  extra step: after each update to the channel, Alice gives Bob a
+  signature over the current state (which includes a state number).
+  Then, each time Alice organically reconnects to Bob, she requests the
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-23-newsletter.md:58]:
```suggestion
  Then, each time Alice organically reconnects with Bob, she requests the
```

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    is for Alice to just download the latest state from Bob each time she
+    organically reconnects to him with no fraud proofs involved.  If he
+    provides her with an old state, she closes the channel with him,
+    denying him any further forwarding fees he would earn from her
+    future payments.  This sounds very similar to the version of [peer
+    storage][topic peer storage] described in [BOLTs #881][], the
+    version experimentally implemented in Core Lightning earlier this
+    year (see [Newsletter #238][news238 peer storage]), and (according
+    to a message from Bastien Teinturier [indicates][teinturier
+    backups]) a verson of the scheme implemented in the Phoenix wallet
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-23-newsletter.md:87]:
```suggestion
    backups]) a version of the scheme implemented in the Phoenix wallet
```

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+- [Bitcoin Core #27460][] rpc: Add importmempool RPC FIXME:Murchandamus
+
+- [Bitcoin Core #27981][] fixes a bug that could have potentially caused
+  two nodes to be unable to receive data from each other.  If Alice's
+  node had a lot of data queued to send to Bob's node, she would try to send
+  that data before accepting new data from Bob.  If Bob also had a lot
+  of data queued to send to Alice's node, he also wouldn't accept new data
+  from Alice.  This could lead to neither of them attempting to receive
+  data from the other indefinitely.  The issue was original discovered
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-23-newsletter.md:150]:
```suggestion
  data from the other indefinitely.  The issue was originally discovered
```

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 4]
```diff
@@ -174,7 +174,9 @@ Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
 Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
 [Bitcoin Inquisition][bitcoin inquisition repo].*
 
-- [Bitcoin Core #27460][] rpc: Add importmempool RPC FIXME:Murchandamus
+- [Bitcoin Core #27460][] adds a new `importmempool` RPC. The RPC will
+  load a `mempool.dat` file and attempt to add the loaded transactions
+  to its mempool.
```
Comment (murchandamus) [inline _posts/en/newsletters/2023-08-23-newsletter.md:179]: I went back and forth with myself, whether it is useful to mention the options to the broader audience, but I didn’t have a good idea how to make it relevant. My closest attempt was to say:

“The RPC has options to import some transaction metadata from the imported file that may be useful to mining pool operators or other power users.”

But I was only half convinced. Any thoughts?

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+- **Blixt Wallet v0.6.8 released:**
+  The [v0.6.8 release][blixt v0.6.8] includes support for [hold invoices][topic
+  hold invoices] and [zero-conf channels][topic zero-conf channels] among other improvements.
+
+- **Sparrow 1.7.8 released:**
+  Sparrow [1.7.8][sparrow 1.7.8] added support for [BIP322][] [message signing][topic generic
+  signmessage], including P2TR addresses, and made various improvements to
+  [RBF][topic rbf] and [CPFP][topic cpfp] fee bumping features.
+
```
Comment (MarnixCroes) [inline _posts/en/newsletters/2023-08-23-newsletter.md:139]:
```suggestion

- **Wasabi Wallet 2.0.4 released:**
  Wasabi [2.0.4][wasabi 2.0.4] includes a daemon, TurboSync (for faster wallet load), 
  privacy suggestions when sending, a _Speed Up Transaction_ button for increasing the
  fee of an unconfirmed transaction with [RBF][topic rbf] or [CPFP][topic cpfp], a new 
  amount decomposer for the coinjoins and lots of other improvements and bug fixes.
```

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+[teinturier backups]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004045.html
+[ghost43 backups]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004052.html
+[voegtlin backups2]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-August/004055.html
+[Scaling Lightning]: https://github.com/scaling-lightning/scaling-lightning
+[sl twitter update]: https://twitter.com/max_blue__/status/1681781001373065216
+[sl tg]: https://t.me/+AytRsS0QKH5mMzM8
+[torq github]: https://github.com/lncapital/torq
+[torq blog]: https://ln.capital/articles/announcing-torq-V1.0
+[blixt v0.6.8]: https://github.com/hsjoberg/blixt-wallet/releases
+[sparrow 1.7.8]: https://github.com/sparrowwallet/sparrow/releases/tag/1.7.8
```
Comment (MarnixCroes) [inline _posts/en/newsletters/2023-08-23-newsletter.md:239]:
```suggestion
[sparrow 1.7.8]: https://github.com/sparrowwallet/sparrow/releases/tag/1.7.8
[wasabi 2.0.4]: https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4
```

**Review by MarnixCroes — COMMENTED** [review]
new Wasabi release yesterday https://github.com/zkSNACKs/WalletWasabi/releases/tag/v2.0.4

File: _posts/en/newsletters/2023-08-23-newsletter.md (hunk 5 above)
Comment (bitschmidty) [reply _posts/en/newsletters/2023-08-23-newsletter.md:139]: @MarnixCroes thanks for the heads up. Reworded a bit and added here: https://github.com/bitcoinops/bitcoinops.github.io/pull/1250/commits/c3d8c4a97f756b0bd506dbda49c66103bc9c6a39

File: _posts/en/newsletters/2023-08-23-newsletter.md (hunk 4 above)
Comment (glozow) [reply _posts/en/newsletters/2023-08-23-newsletter.md:184]: fwiw I don't really see those options being useful to someone who isn't a power user + trying to replicate something very specific on a test node

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27460][] adds a new `importmempool` RPC. The RPC will
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-08-23-newsletter.md:182]: This might be helpful for readers as well
https://bitcoincore.reviews/27460

**Comment by bitschmidty** [comment]
Note: Travis CI is not running and Im working with support to get this resolved.

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+slug: 2023-08-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Fraud proofs for outdated backup state:** Thomas Voegtlin
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2023-08-23-newsletter.md:14]: Leaving as-is.

File: _posts/en/newsletters/2023-08-23-newsletter.md (hunk 4 above)
Comment (harding) [reply _posts/en/newsletters/2023-08-23-newsletter.md:184]: If y'all are unconvinced, I'm unconvinced.  Dropping it from this week's newsletter.  If anyone has second thoughts, we can always cover it next week.

**Comment by harding** [comment]
Addressed review comments (thanks @LarryRuane @murchandamus @glozow !), reviewed contributions from @bitschmidty and @murchandamus (thanks!), added lede, releases/RCs, and topic entries.  Also rand the production tests locally (they passed).

THanks everyone!

File: _posts/en/newsletters/2023-08-23-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+title: 'Bitcoin Optech Newsletter #265'
+permalink: /en/newsletters/2023/08/23/
+name: 2023-08-23-newsletter
+slug: 2023-08-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes fraud proofs for outdated backup state
+and includes our regular sections summaring recent changes to services
```
Comment (azuchi) [inline _posts/en/newsletters/2023-08-23-newsletter.md:11]:
```suggestion
and includes our regular sections summarizing recent changes to services
```

Comment (bitschmidty) [reply _posts/en/newsletters/2023-08-23-newsletter.md]: Thanks for catching this @azuchi !

**Comment by bitschmidty** [comment]
CI fixed. 
Pushed a fixup for @azuchi feedback (https://github.com/bitcoinops/bitcoinops.github.io/pull/1250#discussion_r1302323580) and a topic wording fixup.
Squashed and merged.

Thank you @harding @LarryRuane @murchandamus @MarnixCroes @azuchi @glozow for contributing this week! 🚀

File: _posts/en/newsletters/2023-08-23-newsletter.md (hunk 4 above)
Comment (glozow) [reply _posts/en/newsletters/2023-08-23-newsletter.md]: Ah just to clarify, I didn't mean that the summary should be dropped, just that a description of the options didn't need to be added. I think the RPC in general is worth covering!

**Pull request merged**